project/requirements.md

```diff
@@ -0,0 +1,20 @@
+### Final Project: Calculator
+
+#### Due Date
+A pull request with your code must be issued to the unit-1 `calculator` branch by 18:59 on 5/12 (that is, before class). Presentations will be in class that day.
+
+#### Groups
+You will work in assigned [groups of two](groups.md).
+
+#### Description
+
+Create a calculator Android application. This application should be both well-designed and functional. Feel free to add additional features beyond the requirements of the application. When submitting your project, include a breakdown of what each group member did (and be prepared to talk about this during the demo) and include a README.md which explains the overall structure of your code.
+
+#### Requirements
+* Handle user input and display the results.
```

Comment:
I think the fact that they're creating a scientific calculator could be better emphasized by adding "scientific" to the project title. I fear that since it is mentioned at the very end of the project description, the devs will probably think that the scientific aspect of the calculator is a "nice to have" feature and perhaps not part of the core project requirement. This is at least how I interpret it.